.travis.yml

```diff
@@ -8,8 +8,10 @@ php:
   - 5.4
 
 install:
-  - composer install --dev --prefer-source
+  - composer install --prefer-source
 
 script:
   - bin/phpunit
   - bin/psecio-parse scan src tests
+
+dist: trusty
```

Comment:
Just trying to avoid massive changes, like changing used Ubuntu version for all builds just because HHVM one fails. You can just move `dist:` part up as I've originally posted. That would be fine by me as well.